m3u8dl/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .core.m3u8dl import main
```

Comment:
Codacy found an issue: [Missing docstring in public package (D104)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6280464)